dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -86,6 +86,7 @@
   private static String simulateStreamScriptDestinationFolder = TestConfig.tab().stringAt(SnappyPrms.simulateStreamScriptDestinationFolder, dtests);
   public static boolean isLongRunningTest = TestConfig.tab().booleanAt(SnappyPrms.isLongRunningTest, false);  //default to false
   public static boolean isUserConfTest = TestConfig.tab().booleanAt(SnappyPrms.isUserConfTest, false);  //default to false
+  public static boolean isCppTest = TestConfig.tab().booleanAt(SnappyPrms.isCppTest, false);  //default to false
```

Comment:
Why are there so many formatting changes in this file. Have you made change in this file?

I will also check. I think you should revert formatting changes in this file. Let QA folks handle that?

Reply:
Reverted formatting changes